### PR TITLE
feat(dev): Add basic environment and requests for IDEA HTTP client

### PR DESCRIPTION
**Pull request by MarcelBochtler**
Add requests to read and create the basic ORT server entities with the IDEA HTTP client [1].
This setup expects a `http-client.private.env.json` file that configures the authentication.

[1]: https://www.jetbrains.com/help/idea/http-client-in-product-code-editor.html

### CODE REVIEW

**Comment by MarcelBochtler** [comment]
Discussion:
I added only a very basic number of requests to keep it simple.
We could also import all the requests from our OpenApi spec. But I'm not sure if this would add too many and will make it more complex.

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,9 @@ out/
 # Helm subcharts and install packages
 /scripts/helm/ort-server/charts/
 /scripts/helm/*.tgz
+
+# Private environment for IDEA HTTP client
+http-client.private.env.json
+
+# Private http requests
```
Comment (mnonnenmacher) [inline .gitignore:20]: Nit: HTTP (as in comment above)

**Review by mnonnenmacher — CHANGES_REQUESTED** [review]
Could you add a README to the `requests` dir that briefly explains what these files are about and than IntelliJ Ultimate license is required to use them?

**Comment by mnonnenmacher** [comment]
> Discussion: I added only a very basic number of requests to keep it simple. We could also import all the requests from our OpenApi spec. But I'm not sure if this would add too many and will make it more complex.

I think it's good to start with a small number of requests to keep it maintainable.

**Review by mnonnenmacher — APPROVED** [review]

**Pull request merged**